libquickevent/libquickeventgui/src/reportoptionsdialog.h

```diff
@@ -38,7 +38,7 @@ class QUICKEVENTGUI_DECL_EXPORT ReportOptionsDialog : public QDialog, public qf:
 	QF_PROPERTY_BOOL_IMPL2(v, V, acantsVisible, true)
 	QF_PROPERTY_BOOL_IMPL2(s, S, tagesOptionVisible, false)
 	QF_PROPERTY_BOOL_IMPL2(l, L, egsOptionVisible, false)
-	QF_PROPERTY_BOOL_IMPL2(p, P, ageLayoutVisible, false)
+	QF_PROPERTY_BOOL_IMPL2(p, P, ageLayoutVisible, true)
```

Comment:
jak tohle souvisi s exportem startovky?

Reply:
``grpPageLayout`` je defaultne zobrazen (visible je true)
Promenna ``pageLayoutVisible`` byla v pocatecnim stavu false
Byla nekonzistence a pokud nekdo zavolal ``setPageLayoutVisible(false);`` tak se nic nestalo.
Oprava to sjednotila.

Reply:
Nesouvisi to, jenom jsem to pri psani teto feature objevil a opravil.